tests/integrated/test-interp_to/README.md

```diff
@@ -0,0 +1,7 @@
+test-interp_to
```

Comment:
I wonder if this may be better as a unit test?

Reply:
Could well be, but I'm afraid I haven't learned to write unit tests yet... Any offers to write one welcome :wink:

Reply:
I can have a look, but may not get chance for a while. The existing unit tests give some examples that can usually be modified as a starting point. I'll ping @ZedThree as well as he may be able to help.